apps/08jquery-towers-of-hanoi/script.js

```diff
@@ -1,5 +1,35 @@
 'use strict';
 
+var $block = null;
 $(document).ready(function() {
-  // Put app logic here
+  $('[data-stack]').on('click', function() {
+    //This is were the blocks will be moved from one stack to another stack
```

Comment:
I think this could be a little more specific; I suggest breaking into two comments: one each for 'pick up' and 'put down'